src/components/dds/Notification/Notification.tsx

```diff
@@ -0,0 +1,136 @@
+import getRootClassNames from '../common/getRootClassNames';
+import { componentClasses } from 'dds-core/components/notification/notification';
+import type { ButtonProps } from '../Button';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+import type { Icons } from 'dds-core/components/icon/ts/icon';
+import 'dds-core/components/notification/_notification.scss';
+
+export interface NotificationProps extends React.HTMLAttributes<HTMLDivElement> {
+    /**
+     * Title text for the notification
+     * */
+    title: string;
+    /**
+     * Message of the notification
+     * */
+    message: ReactNode;
+    /**
+     * Timestamp for the notification
+     * */
+    timestamp: string;
+    /**
+     * Whether the close icon should be visible
+     * */
+    closeIcon: boolean;
+    /**
+     * Icon name for the notification
+     * */
+    titleIcon: Icons;
+    /**
+     * Main notification button props
+     *
+     * If not specified, the button will not be displayed
+     */
+    primaryButton?: ButtonProps;
+    /**
+     * Secondary notification button props
+     *
+     * If not specified, the button will not be displayed
+     */
+    secondaryButton?: ButtonProps;
+    /**
+     * Whether the notification is visible
+     * */
+    open?: boolean;
+    /**
+     * Action executed on close button click
+     * */
+    onClose?: () => void;
+}
+
+const StyledCloseButton = styled(Button)`
+    padding: 0;
+    font-size: 1em;
+    line-height: 1;
+`;
+const StyledIcon = styled(Icon)`
+    && {
+        margin-right: 0;
+        font-size: 1em;
+    }
+`;
+
+/**
+ * React implementation of DDS notification component
+ */
+export function Notification({
+    className,
+    title,
+    message,
+    timestamp,
+    closeIcon = true,
+    titleIcon,
+    primaryButton,
+    secondaryButton,
+    open,
+    onClose,
+    ...rest
+}: NotificationProps) {
+    const [visible, setVisible] = React.useState(open);
+
+    React.useEffect(() => {
+        setVisible(open);
+    }, [open]);
+
+    const handleClose = () => {
+        setVisible(false);
+        if (onClose) {
+            onClose();
+        }
+    };
+    return (
+        <div
+            className={getRootClassNames(className, componentClasses.base, visible && componentClasses.show)}
+            role="alert"
+            aria-live="polite"
+            aria-atomic="true"
+            aria-hidden={!visible}
+            {...rest}
+        >
+            <Icon name={titleIcon} className={componentClasses.icon} />
+            <div className={componentClasses.body}>
+                <div className={componentClasses.header}>
+                    <span className={componentClasses.headerTitle}>{title}</span>
+                    {closeIcon && (
+                        <StyledCloseButton
+                            variant="tertiary"
+                            className={classNames(componentClasses.button.base, componentClasses.closeButton)}
```

Comment:
We don't need to set `componentClasses.button.base` here - it's already set by `Button`.
```suggestion
                            className={componentClasses.closeButton}
```